bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastBindingConstants.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yamahamusiccast.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link YamahaMusiccastBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Lennert Coopman - Initial contribution
+ */
+@NonNullByDefault
+public class YamahaMusiccastBindingConstants {
+
+    private static final String BINDING_ID = "yamahamusiccast";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_DEVICE = new ThingTypeUID(BINDING_ID, "Device");
```

Comment:
Thing Type UIDs should start with lower case.

Reply:
Done, available with last push